docs/03-tutorials/00-api-exposure/apix-03-expose-and-secure-workload-oauth2.md

```diff
@@ -0,0 +1,246 @@
+---
+title: Expose and secure a workload with OAuth2
+---
+
+This tutorial shows how to expose and secure services or Functions using API Gateway Controller. The controller reacts to an instance of the API Rule custom resource (CR) and creates an Istio Virtual Service and [Oathkeeper Access Rules](https://www.ory.sh/docs/oathkeeper/api-access-rules) according to the details specified in the CR. To interact with the secured services, the tutorial uses an OAuth2 client registered through the Hydra Maester controller.
+
+The tutorial may be a follow-up to the [Use a custom domain to expose a workload](./apix-01-own-domain.md) tutorial.
+
+## Prerequisites
+
+This tutorial is based on a sample HttpBin service deployment and a sample Function. To deploy or create one of those, follow the [Create a workload](./apix-02-create-workload.md) tutorial.
+
+## Register an OAuth2 client and get tokens
+
+1. Export your client as an environment variable:
+
+   ```shell
+   export CLIENT_NAME={YOUR_CLIENT_NAME}
+   ```
+
+2. Create an OAuth2 client with `read` and `write` scopes. Run:
```

Comment:
Later in the document, you use quotation marks in scopes formatting ("read" and "write"). Maybe it would be good to unify that?